README.md

```diff
@@ -81,7 +92,15 @@ curl --request POST \
 ```
 
 ```csharp
-// Coming Soon
+ApiCreateCallRequest callRequest = new ApiCreateCallRequest();
```

Comment:
Can you add the proper `using` statements like you have for messaging?

Reply:
For brevity sake, we can leave out initialization from the Readme snips